package.json

```diff
@@ -206,7 +206,13 @@
                     "type": "boolean",
                     "description": "When there is a previous search pattern, highlight all its matches.",
                     "default": false
+                },
+                "vim.autoindent": {
+                    "type": "boolean",
+                    "description": "Keep indentation when doing cc or S in normal mode to replace a line.",
```

Comment:
we'd better use Vim's official description, see http://vimdoc.sourceforge.net/htmldoc/options.html#'autoindent'